schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BIT, 
-    weight DECIMAL
+    weight DECIMAL,
+    species varchar(100);
```

Comment:
- [OPTIONAL] Adding a new column to a table directly is not a good approach.  Let's assume you have some data in your table and for adding a new column you need to delete the existing table and create the table with the new column. Therefore, it is good to always alter a table and update the new column for the records.

See example :
```sql 
ALTER TABLE animals ADD species varchar(255);
```